test/integration/apiserver/admissionwebhook/admission_test.go

```diff
@@ -323,25 +325,20 @@ func TestWebhookV1beta1(t *testing.T) {
 	defer webhookServer.Close()
 
 	// start API server
-	master := etcd.StartRealMasterOrDie(t, func(opts *options.ServerRunOptions) {
-		// turn off admission plugins that add finalizers
-		opts.Admission.GenericAdmission.DisablePlugins = []string{"ServiceAccount", "StorageObjectInUseProtection"}
-
-		// force enable all resources so we can check storage.
-		// TODO: drop these once we stop allowing them to be served.
-		opts.APIEnablement.RuntimeConfig["extensions/v1beta1/deployments"] = "true"
-		opts.APIEnablement.RuntimeConfig["extensions/v1beta1/daemonsets"] = "true"
-		opts.APIEnablement.RuntimeConfig["extensions/v1beta1/replicasets"] = "true"
-		opts.APIEnablement.RuntimeConfig["extensions/v1beta1/podsecuritypolicies"] = "true"
-		opts.APIEnablement.RuntimeConfig["extensions/v1beta1/networkpolicies"] = "true"
-	})
-	defer master.Cleanup()
+	s, err := kubeapiservertesting.StartTestServer(t, kubeapiservertesting.NewDefaultTestServerOptions(), []string{
```

Comment:
StartRealMasterOrDie also sets up test CRDs:

```go
	// create CRDs so we can make sure that custom resources do not get lost
	CreateTestCRDs(t, apiextensionsclientset.NewForConfigOrDie(kubeClientConfig), false, GetCustomResourceDefinitionData()...)
```

Reply:
what for?

Reply:
I would prefer standard plumbing and explicit fixtures than something hidden deep in custom plumbing code.

Reply:
no disagreement, we can hoist that fixture setup out to the places that require it. it grew organically and all the tests using that setup method intend to exercise an operation (checking etcd storage path, dry-run, or webhook admission) over all resources, including custom resources

Reply:
fixed